rinja/src/filters/builtin.rs

```diff
@@ -93,6 +169,31 @@ pub fn paragraphbreaks(s: impl fmt::Display) -> Result<HtmlSafeOutput<String>, f
 }
 
 /// Converts to lowercase
+///
+/// ```
+/// /// ```
```

Comment:
I just realized, why is there this line in this code example and below? I think it's a typo because the doctest is supposed to start at <code>```jinja</code>.